### PR TITLE
feat/generate_dbcontext

### PR DESCRIPTION
**Pull request by HaiYenNguyen12**

### CODE REVIEW

File: HotelBooking.Data/Infrastructure/IRepository.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HotelBooking.Data.Infrastructure
+{
+    internal interface IRepository
```
Comment (orienthoanguyen) [inline HotelBooking.Data/Infrastructure/IRepository.cs:9]: make it public

File: HotelBooking.Data/Infrastructure/IUnitOfWork.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HotelBooking.Data.Infrastructure
+{
+    internal interface IUnitOfWork
```
Comment (orienthoanguyen) [inline HotelBooking.Data/Infrastructure/IUnitOfWork.cs:9]: make it public

File: HotelBooking.API/appsettings.json [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
-      "Microsoft.AspNetCore": "Warning"
+    "Logging": {
+        "LogLevel": {
+            "Default": "Information",
+            "Microsoft.AspNetCore": "Warning"
+        }
+    },
+    "AllowedHosts": "*",
+    "ConnectionStrings": {
+        "Default": "Server=DD-YENNTH1\\SQLEXPRESS;Database=HotelBooking;Trusted_Connection=True;TrustServerCertificate=True;"
```
Comment (orienthoanguyen) [inline HotelBooking.API/appsettings.json:10]:
```suggestion
        "Default": "Server=.\SQLEXPRESS;Database=HotelBooking;Trusted_Connection=True;TrustServerCertificate=True;"
```
.\SQLEXPRESS will work for all locally.

Comment (HaiYenNguyen12) [reply HotelBooking.API/appsettings.json:10]: Do you mean .\\SQLEXPRESS?

**Review by orienthoanguyen — APPROVED** [review]

**Pull request merged**